src/cgame/cg_rocket_draw.cpp

```diff
@@ -1029,6 +1029,34 @@ class WallwalkElement : public HudElement
 	bool isActive;
 };
 
+class SprintElement : public HudElement
+{
+public:
+	SprintElement( const Rocket::Core::String& tag ) :
+			HudElement( tag, ELEMENT_HUMANS ),
+			isActive( false ) {}
+
+	void DoOnUpdate()
+	{
+		bool sprinting = cg.snap->ps.stats[ STAT_STATE ] & SS_SPEEDBOOST;
+		if ( ( sprinting && !isActive ) || ( !sprinting && isActive ) )
+		{
+			SetActive( sprinting );
+		}
+	}
+
+private:
+	void SetActive( bool active )
+	{
+		isActive = active;
+		SetClass( "active", active );
```

Comment:
The class should be set from the beginning instead of only after the first time the user sprints.